src-tauri/capabilities/default.json

```diff
@@ -24,8 +26,22 @@
     "http:allow-fetch-send",
     {
       "identifier": "http:default",
-      "allow": [{ "url": "https://*.tauri.app" }],
-      "deny": [{ "url": "https://private.tauri.app" }]
-    }
+      "allow": [
+        {
+          "url": "http://localhost:2900"
+        },
+        {
+          "url": "https://*.tauri.app"
```

Comment:
what is https://*.tauri.app used for?

Reply:
Don't remove it for now. Removing it requires additional configuration. Removing https://private.tauri.app is feasible, but make sure the CSP settings meet your application requirements and maintain appropriate security. This is a configuration that comes with the Tauri framework

Reply:
https://*.tauri.app has been deleted